pkgs/by-name/qu/quarkus/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchurl
+, writeScript
+, bash
+, openjdk
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "quarkus-cli";
+  version = "3.8.3";
+
+  src = fetchurl {
+    url = "https://github.com/quarkusio/quarkus/releases/download/${finalAttrs.version}/quarkus-cli-${finalAttrs.version}.tar.gz";
+    hash = "sha256-+HUVnd21q2xln58UR1QS0UFCjlY0sbf79ZkvWbRCH7Q=";
+  };
+
+  launcher = writeScript "quarkus" ''
+    #! ${bash}/bin/bash
+
+    export CLASSPATH=@LIB@/quarkus-cli-${finalAttrs.version}-runner.jar
```

Comment:
```suggestion
    export CLASSPATH=@out@/lib/quarkus-cli-${finalAttrs.version}-runner.jar
```

Reply:
well, you reviewed it before my next push aha